_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Imagine—if in portrait mode, the video plays right in the video details page. Raise your hands and say "the day has come". Believe.
+NewPipe now comes equipped with a setting for 'Autoplay'. It can be enjoyed responsibly for automatic video play upon opening video details.
+Such lavishness is of course off by default for mobile data, but works out of the box provided Wi-Fi compatible equipment is within reach.
 
 ### Video orientation
 
-Previously, the main video player would always play videos in full-screen regardless of your selected orientation. This was fine for landscape videos in landscape mode and portrait videos in portrait mode, but it resulted in a lot of wasted space if the video's orientation didn't match the player's. No more! There is now a full-screen button which replaces the orientation switch button, and it is _smart_. It will detect if the video is landscape or portrait, and will ensure the player is in the correct orientation when you enter full-screen! How's _that_ for convenience and usability? Not only that, but if auto-rotate is enabled, and the video is a landscape one, the button will disappear because you don't need it! (In portrait mode, you still need the button to go into full-screen from the video details.)
+In the times of old, the main video player would always play videos full-screen regardless of the user's orientation. Great for video in the realm of wider than tall, if you were holding it right.
+Somehow also great if held upright for perusers of vertical-video. Or, great for any user unconcerned with 90° viewing angles in either direction.
+In the glory that is NewPipe 0.20.0, a full-screen button replaces the orientation-switch button. Is the video a good old-fashioned video, or some vertical nonsense? The player knows, and the player checks to see if the device is held the same way before going full-screen. If you have auto-rotate on, the button vanishes for regular videos, but will be to go from vertical video details, to full vertical video on the vertical. If you are into that sort of thing.
 
 <video src="/img/smart_button.webm" poster="/img/0.20.0_thumbnail_smart_fullscreen.png" class="img-responsive no-flow screenshot-image" autoplay loop muted controls></video>
 
 ### Minimized player
 
-You can now swipe down on the video in the video details page to _minimize_ it - just like YouTu... wait no. It's better. You get a minimized player at the bottom, and you can play/pause or even close it from there. How cool is that? The video will keep playing (or stay paused, if you prefer that) while you browse NewPipe at your leisure! No more having to finish a video before browsing mindlessly! Yay distractions! (In landscape mode, you can swipe down on the title to minimize as well. The swipe area is confined to that region because the rest is used for the existing volume and brightness gestures, as you know.)
```

Comment:
"Minimize it" is the end statement in a garden-path sentence that reads "you can now swipe down on the video". That should be downwards. Of course it is now. Stick to explaining one thing, and then do everything else as "in the past".

Reply:
> "Minimize it" is the end statement in a garden-path sentence

Your usage of the term "garden-path sentence" is incorrect. See the Wikipedia page: https://en.wikipedia.org/wiki/Garden-path_sentence

> That should be downwards.

Swipe up, swipe down, swipe left, swipe right - they are all commonly used terms. I have never read "swipe upwards", "swipe leftwards", etc. anywhere.

> Stick to explaining one thing, and then do everything else as "in the past".

Why does it matter if we say "This is what happened earlier; this is what happens now" vs "This is what happens now; this is what happened earlier"?

> Why does everything hinge on something else. 

Explain? Or an example?

> What region.

The title region. It is mentioned in the previous sentence.

> There is no "as you know".

It is a reasonable assumption to me that the user knows and uses the volume and brightness gestures.

Reply:
Why are we talking to the user about things the user knows already? I have been using NewPipe since it was created, and I don't do any swiping. I had swiping over a decade ago in Opera, and I still didn't use it. It isn't reasonable, it is counterfactual.

Time matters, because there is no voice established, no narrative. It just lumps on extra detail for no reason. Like adding "you" to everything that can be done.



>>Why does everything hinge on something else.

>Explain? Or an example?



>What region.

>>The title region. It is mentioned in the previous sentence.

The swiping is a question of what is the swiping action, and what is the swiping result. I didn't make it clear.

"You can now swipe down on the video in the video details page to" to what? Why?